_assets/ci/update-geth.sh

```diff
@@ -12,7 +12,7 @@ if [ ! -z "$GETH_BRANCH" ]; then
 	sed -i 'N;N;s@\(\[\[constraint]]\n  name = "github.com\/ethereum\/go-ethereum"\n  branch =\)\(.*\)@\1 '"\"${GETH_BRANCH}\""'@g' Gopkg.toml
 fi
 dep ensure -v -update github.com/ethereum/go-ethereum
-make dep-ensure || (echo "Please fix patches. (dep-ensure failed)" && exit 1)
+make dep-ensure || (echo "Please fix patches and rerun. (dep-ensure failed)" && exit 1)
```

Comment:
Interestingly, I wanted to make this `if $(make dep-ensure); then...` (similar to what we did for `git diff` below), but was running into a peculiar error when I had it set up this way: 

```Bash
$ make update-geth
./_assets/ci/update-geth.sh 
make[1]: *** [dep-ensure] Error 1
./_assets/ci/update-geth.sh: line 14: make[1]:: command not found
```

Anyway, I tested just doing `make dep-ensure || (echo...`  and it does the trick in the positive and negative case:

Positive
-
```
make update-geth                                                                            
./_assets/ci/update-geth.sh swog
make[1]: Entering directory `/home/alex/workspace/src/github.com/status-im/status-go'
...
#@dep ensure
#./_assets/patches/patcher
make[1]: Leaving directory `/home/alex/workspace/src/github.com/status-im/status-go'
Geth updated.
```

Negative
-
```Bash
$ make update-geth
./_assets/ci/update-geth.sh 
make[1]: Entering directory `/home/alex/workspace/src/github.com/status-im/status-go'
...
#@dep ensure
#./_assets/patches/patcher
make[1]: *** [dep-ensure] Error 1
make[1]: Leaving directory `/home/alex/workspace/src/github.com/status-im/status-go'
Please fix patches and rerun. (dep-ensure failed)
make: *** [update-geth] Error 1
```

Reply:
Ah, figured it out - didn't need the parameter expansion.